### PR TITLE
add the news model

### PR DESCRIPTION
**Pull request by deepakrana0011**

### CODE REVIEW

File: lib/core/db_models/old/schema/table_collection_schema.dart [hunk 1]
```diff
@@ -26,4 +26,5 @@ class FbCollections {
   static const String checks = "checks";
   static const String occupations = "occupations";
   static const String workHistories = "work_histories";
+  static const String workerNotificationMessage = "worker_notification_messages";
```
Comment (SamDuncanAsomas) [inline lib/core/db_models/old/schema/table_collection_schema.dart:29]: guessing this is a mistake and we shouldn't have an underscore between these words

File: lib/core/db_models/worker/schema/newsNotificationSchema.dart [hunk 2]
```diff
@@ -0,0 +1,15 @@
+class NewsNotificationSchema {
```
Comment (SamDuncanAsomas) [inline lib/core/db_models/worker/schema/newsNotificationSchema.dart:1]: guessing this is a mistake and we shouldn't have an underscore between these words

**Review by SamDuncanAsomas — CHANGES_REQUESTED** [review]

File: lib/core/db_models/old/schema/table_collection_schema.dart (hunk 1 above)
Comment (deepakrana0011) [reply lib/core/db_models/old/schema/table_collection_schema.dart:29]: sam this is the collection name and it should be same as mentioned in the firestore

File: lib/core/db_models/worker/schema/newsNotificationSchema.dart (hunk 2 above)
Comment (deepakrana0011) [reply lib/core/db_models/worker/schema/newsNotificationSchema.dart:1]: sam these are the keys  return from the api and it should be same as mentioned in the api to parse the data

**Pull request merged**